templates/mapstore/mapstore-deployment.yaml

```diff
@@ -1,5 +1,5 @@
-{{- if .Values.georchestra.webapps.mapstore.enabled -}}
-{{- $webapp := .Values.georchestra.webapps.mapstore -}}
+{{- if $webapp.mapstore.enabled -}}
+{{- $webapp := $webapp.mapstore -}}
```

Comment:
duh ?
Defined before use in line 1. and this line 2 looks like a search'n'replace gone wild

Reply:
resolved